index.bs

```diff
@@ -6668,7 +6674,7 @@ In effect, where <var ignore>x</var> is a <emu-val>Number</emu-val> value,
 
     1.  Let |x| be [=?=] [=ToNumber=](|V|).
     1.  If |x| is <emu-val>NaN</emu-val>, +∞, or −∞,
-        then [=ECMAScript/throw=] a <emu-val>TypeError</emu-val>.
+        then [=ECMAScript/throw=] a {{TypeError}}.
```

Comment:
Yes, this was intentional. TypeError [maps](https://heycam.github.io/webidl/#es-exception-objects) to ES anyway, while SyntaxError is a special case as it's an example of an exception type that doesn't map to IDL.